### PR TITLE
Deprecate in favour of puppetlabs-concat 2.x

### PR DESCRIPTION
**Pull request by domcleal**

### CODE REVIEW

**Comment by ekohl** [comment]
:+1:

File: README.md [hunk 1]
```diff
@@ -8,6 +8,17 @@ Fragment Pattern.
 The concept is based on ideas that R.I. Pienaar describes on his [Building
 files from fragments in Puppet](http://www.devco.net/archives/2010/02/19/building_files_from_fragments_with_puppet.php) page.
 
+Deprecation warning
+-------------------
+
+This module is deprecated in favour of [puppetlabs-concat](http://forge.puppetlabs.com/puppetlabs/concat)
```
Comment (mmoll) [inline README.md]: https there would be better, IMHO.

Comment (domcleal) [inline README.md]: Fixed.

**Comment by mmoll** [comment]
thanks @domcleal!

**Comment by domcleal** [comment]
Released as 1.5.0 as per semver.

**Pull request merged**